pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -420,7 +420,7 @@ protected CompletableFuture<Void> internalCreateNonPartitionedTopicAsync(boolean
             .thenCompose(partitionedTopicMetadata -> {
                 int currentMetadataPartitions = partitionedTopicMetadata.partitions;
                 if (currentMetadataPartitions <= 0) {
-                    throw new RestException(422 /* Unprocessable entity*/,
+                    throw new RestException(Status.CONFLICT /* Unprocessable entity*/,
```

Comment:
@mattisonchao - the Javadoc for this method and for the methods that call it are now out of sync. Would you mind following up and updating them?